content/posts/2021/q3-update/index.md

```diff
@@ -0,0 +1,91 @@
+
+### JupyterHub community support
+
+We recently collaborated with the JupyterHub community to [apply for two CZI EOSS awards](https://github.com/jupyterhub/team-compass/issues/380). Last month, we were notified that [our application to support a Community Strategic Lead](https://chanzuckerberg.com/eoss/proposals/jupyterhub-community-strategic-lead/) was funded! This role will fund Sarah Gibson's time to focus some of her thinking on building community structures and dynamics that are inclusive and sustainable. We'll update with more information as this project starts moving.
+
+### Organizational foundations
+
+Finally, in addition to our major development and projects, we have also made a lot of progress at an organizational level.
+
+We **began [a fiscal sponsorship with Code for Science and Society](https://2i2c.org/posts/2021/css-announce/)**. This provides a new organizational and legal home for 2i2c after spending nearly a year receiving [critical strategic and start-up support](https://www.icsi.berkeley.edu/icsi/news/2021/08/2i2c-new-chapter) from our previous host, [ICSI](https://icsi.berkeley.edu). We are excited to work with CS&S to create the business infrastructure that will power our managed JupyterHubs service. (more on that later)
+
+The 2i2c team has also been **improving our team planning and coordination processes**, so that we can more effectively execute on our mission. As a distributed team, we have the challenge of building processes for team communication, coordination, and planning that are distributed and asynchronous. If you're curious, you can learn more about our coordination processes in [our Team Compass](https://team-compass.2i2c.org/en/latest/practices/development.html).
+
+We have **improved our organization-wide documentation** in order to make it easier to navigate between 2i2c's various sources of information. We hope that this provides more transparency into what 2i2c is up to and how it is structured, and that it allows us to build more connections between our projects and the broader community. Check out the new documentation landing site at [docs.2i2c.org](https://docs.2i2c.org).
+
+### Open source support
+
+Much of the work above was done in collaboration with upstream open source communities. Three key open source communities that we interact with are the Jupyter, JupyterHub, and Executable Books ecosystems. Over the last quarter, in these communities, 2i2c team members have:
+
+- Opened XXX pull-requests
+- Merged XXX pull-requests
+- Made XXX comments
```

Comment:
I guess you are getting these numbers in some later commit, right?

Reply:
I was trying to decide whether to block this draft on finding these numbers, or just remove this section for now. What do you think?